egs/ljspeech/tts2/run.sh

```diff
@@ -343,7 +219,6 @@ if [ ${stage} -le 6 ] && [ ${stop_stage} -ge 6 ];then
             --n_fft ${n_fft} \
             --n_shift ${n_shift} \
             --win_length "${win_length}" \
-            --iters ${griffin_lim_iters} \
```

Comment:
Is it ok to remove `--iters` option? This is inconsistent with `egs/ljspeech/tts1/run.sh`.

Reply:
In the case of `tts2` recipes, we do not perform griffin lim so it is not needed.

Reply:
I see.

Reply:
Sorry, this is worng.
It uses Grrifin-Lim. I confused with another method...